lib/thumbs/pull_request_worker.rb

```diff
@@ -856,6 +876,53 @@ def comment_code_approvals
       code_reviews
     end
 
+    def parse_thumbot_command(text_body)
+      result_lines = text_body.split(/\n/).grep(/^thumbot/)
```

Comment:
This can will properly parse out multiple commands, i.e. for:
```ruby
text_body = <<EOF
thumbot depends basho-labs/thumbs#53
not a command, just saying thumbs is wicked cool
thumbot depends basho/rfc#411
EOF
```

The remainder of the method only handles the first command. IMO this is okay, but the fact that a single thumbot command per comment should be documented.

Similarly, the fact that `thumbot` must be at first position in the comment should be documented.